src/layers/conv.jl

```diff
@@ -346,6 +346,108 @@ function Base.show(io::IO, l::GATConv)
     print(io, "))")
 end
 
+@doc raw"""
+    GATv2Conv(in => out, σ=identity;
+            heads=1,
+            concat=true,
+            init=glorot_uniform    
+            bias=true, 
+            negative_slope=0.2f0)
+
+GATv2 attentional layer from the paper [How Attentive are Graph Attention Networks?](https://arxiv.org/abs/2105.14491).
+
+Implements the operation
+```math
+\mathbf{x}_i' = \sum_{j \in N(i) \cup \{i\}} \alpha_{ij} W \mathbf{x}_j
+```
+where the attention coefficients ``\alpha_{ij}`` are given by
+```math
+\alpha_{ij} = \frac{1}{z_i} \exp(\mathbf{a}^T LeakyReLU([W \mathbf{x}_i; W \mathbf{x}_j]))
```

Comment:
```suggestion
\alpha_{ij} = \frac{1}{z_i} \exp(\mathbf{a}^T LeakyReLU([W_2 \mathbf{x}_i; W_1 \mathbf{x}_j]))
```

Reply:
on that note. I wanted to add a `share_weights::Bool` parameter to force `W_1 == `W_2`, but as I'm new to Flux I did not know how to put that in exactly. Any pointers?